### ISSUE TITLE
Generate benchmark data

### ISSUE THREAD

**Issue by IAlibay** [openforcefield/protein-ligand-benchmark#65]
Opening a separate issue so we don't lose all the information when #52 gets merged.

Based on discussions the following needs to happen before we move ahead with benchmarking properly:
 - [ ] Finish up the PDB formatting issues with #52
 - [ ] Remove the existing edges.yaml files
 - [ ] Add a new set of edges.yaml using OpenFE's minimum spanning graph planner + Perses mapping
 - [ ] Have both Perses and OpenFE do a quick "will this setup properly" check again at this point
 - [ ] Communicate a shared set of HREX sampler settings
 - [ ] Communicate a shared set of force fields
    - We will likely go with what the current perses defaults are (since those are also the OpenFE defaults)
 - [ ] Communicate a set of targets to start with (eventually we will do them all)
   - OpenFE has started with an initial 6 targets: hif2a, mcl1, p38, thrombin, tnsk2, tyk2
 - [ ] OpenFE will re-run 3x replica, Perses will likely run 1x to double check we get the same results?
 - [ ] Eventually re-run everything once we have a finalised Perses / gufe neq protocol?

**Comment by IAlibay** [comment]
@jchodera just pinging you to see if this plan we came up with works for you? We're working on the idea that OpenFE and Perses are the same underlying code for HREX & vanilla HTF things, so we shouldn't need to duplicate full simulations too much here.
